addons/l10n_fr_invoice_addr/views/account_move_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="account_move_form_l10n_fr_invoice_addr" model="ir.ui.view">
+        <field name="name">l10n_fr_invoice_addr.account.move.form</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='partner_shipping_id']" position="before">
+                <field name="l10n_fr_is_company_french" invisible="1"/>
+            </xpath>
+            <xpath expr="//field[@name='partner_shipping_id']" position="attributes">
+                <attribute name="groups"/>
```

Comment:
that's confusing. Why do you remove groups on partner_shipping_id?

Note that you're doing it for everyone having this module installed, not only users of french companies... in multi companies it may be problematic

Reply:
`groups` needs to be cleared here: this field needs to be accessible for invoices when the company is french no matter the context. I've added a `t-if` now to make it conditional